### PR TITLE
[Docs] Add `dask-ssh` CLI Options and Description.

### PR DESCRIPTION
**Pull request by Beomi**
Add CLI Options from `dask-ssh --help`.

- [v] Tests added / passed
- [v] Passes `flake8 dask`

(Test not needed, editing Docs only.)

### CODE REVIEW

File: docs/source/setup/ssh.rst [hunk 1]
```diff
@@ -24,3 +24,50 @@ Or you can specify a hostfile that includes a list of hosts::
 The ``dask-ssh`` utility depends on the ``paramiko``::
 
     pip install paramiko
+
+
+CLI Options
+-------- 
+
+Launch a distributed cluster over SSH. A 'dask-scheduler' process will run
```
Comment (mrocklin) [inline docs/source/setup/ssh.rst]: You probably wanted to use double backticks here for code highlighting in rst format

    ``dask-scheduler``

File: docs/source/setup/ssh.rst [hunk 2]
```diff
@@ -24,3 +24,50 @@ Or you can specify a hostfile that includes a list of hosts::
 The ``dask-ssh`` utility depends on the ``paramiko``::
 
     pip install paramiko
+
+
+CLI Options
+--------
```
Comment (mrocklin) [inline docs/source/setup/ssh.rst]: Sphinx will give a small warning unless the number of dashes matches the number of characters in the title like the following:

```
CLI Options
-----------
```

File: docs/source/setup/ssh.rst [hunk 3]
```diff
@@ -24,3 +24,50 @@ Or you can specify a hostfile that includes a list of hosts::
...
+-------- 
+
+Launch a distributed cluster over SSH. A 'dask-scheduler' process will run
+on the first host specified in [HOSTNAMES] or in the hostfile (unless
+--scheduler is specified explicitly). One or more 'dask-worker' processes
+will be run each host in [HOSTNAMES] or in the hostfile. Use command line
+flags to adjust how many dask-worker process are run on each host
+(--nprocs) and how many cpus are used by each dask-worker process
+(--nthreads).
+
```
Comment (mrocklin) [inline docs/source/setup/ssh.rst]: I would appreciate adding a small note:

> This table may grow out of date, you should check ``dask-ssh --help`` to get an up-to-date listing of all options.

File: docs/source/setup/ssh.rst [hunk 4]
```diff
@@ -24,3 +24,50 @@ Or you can specify a hostfile that includes a list of hosts::
...
+CLI Options
+-------- 
+
+Launch a distributed cluster over SSH. A 'dask-scheduler' process will run
+on the first host specified in [HOSTNAMES] or in the hostfile (unless
+--scheduler is specified explicitly). One or more 'dask-worker' processes
+will be run each host in [HOSTNAMES] or in the hostfile. Use command line
+flags to adjust how many dask-worker process are run on each host
+(--nprocs) and how many cpus are used by each dask-worker process
+(--nthreads).
```
Comment (mrocklin) [inline docs/source/setup/ssh.rst]: We might also want to include the keywords in code highlighting

    ``--nthreads``

**Comment by Beomi** [comment]
Updated your recommends! thanks for your review :)

**Comment by mrocklin** [comment]
This looks great.  Merging.  Thanks @Beomi !

**Pull request merged**